bundle/manifests/falcon-operator.clusterserviceversion.yaml

```diff
@@ -340,7 +340,7 @@ spec:
           - subjectaccessreviews
           verbs:
           - create
-        serviceAccountName: falcon-operator
+        serviceAccountName: falcon-operator-controller-manager
```

Comment:
I don't understand why the serviceAccountName has changed. Can you please explain, why this is ok?

Reply:
Fixed this.... removed the kustomization changes as well as the service name changes